Source/WebCore/rendering/svg/RenderSVGRoot.cpp

```diff
@@ -315,14 +315,12 @@ void RenderSVGRoot::paintObject(PaintInfo& paintInfo, const LayoutPoint& paintOf
 
     auto adjustedPaintOffset = paintOffset + location();
     if (paintInfo.phase == PaintPhase::Mask && style().visibility() == Visibility::Visible) {
-        // FIXME: [LBSE] Upstream SVGRenderSupport changes
```

Comment:
You can leave it.

Reply:
Yeah, I edited it too quickly, restored.